rules/windows/powershell/powershell_script/posh_ps_using_set_service_to_hide_services.yml

```diff
@@ -0,0 +1,34 @@
+title: Windows PowerShell Web Request
```

Comment:
Is that the correct name?

Reply:
That's what copy-paste does to you when you don't focus 😂 Thanks for the catch. Will fix this.